code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -1,10 +1,8 @@
 //Plasma fire properties
 #define OXYGEN_BURN_RATE_BASE				1.4
 #define PLASMA_BURN_RATE_DELTA				9
-#define PLASMA_UPPER_TEMPERATURE			(1370+T0C)
```

Comment:
hm

Reply:
That was moved to code/__DEFINES/atmospherics.dm, where PLASMA_MINIMUM_BURN_TEMPERATURE was already located.

Same for PLASMA_OXYGEN_FULLBURN